frontend/src/pages/Admin/Resources/readFields.tsx

```diff
@@ -0,0 +1,17 @@
+import { FormikValues } from "formik";
```

Comment:
Why does this function need its own file and why isn't it just an array of objects?

Reply:
This is a type definition not a file..

Reply:
it doesnt but for sake of make code re-usable and easier to read these are broken out, similar to gutenberg edit/save functions.

Reply:
@ASpiteri-BCGov I was referring to the whole readFields.tsx file - which just contains the one function.

@ShawnTurple I think the convention in the case of Gutenberg is one React component - one file. The file extension also changes to indicate a React component(ending in .jsx). Splitting out editFields and readFields(and each of them just return an array of objects) seems a bit arbitrary to me but if you say it works best within the context of this project then I can't argue with that.